node/npm_dependencies.ts

```diff
@@ -1,19 +1,18 @@
 import { promises as fs } from 'fs'
-import { builtinModules, createRequire } from 'module'
+import { builtinModules } from 'module'
 import path from 'path'
 import { fileURLToPath, pathToFileURL } from 'url'
 
 import { resolve, ParsedImportMap } from '@import-maps/resolve'
-import { build, OnResolveResult, Plugin } from 'esbuild'
+import { nodeFileTrace, resolve as nftResolve } from '@vercel/nft'
+import { build } from 'esbuild'
+import getPackageName from 'get-package-name'
 import tmp from 'tmp-promise'
 
-import { nodePrefix, npmPrefix } from '../shared/consts.js'
-
 import { ImportMap } from './import_map.js'
 import { Logger } from './logger.js'
 
-const builtinModulesSet = new Set(builtinModules)
-const require = createRequire(import.meta.url)
+const TYPESCRIPT_EXTENSIONS = new Set(['.ts', '.cts', 'mts'])
```

Comment:
```suggestion
const TYPESCRIPT_EXTENSIONS = new Set(['.ts', '.cts', '.mts'])
```

Reply:
Jesus christ, this one is haunted. Thanks!